blueprints/load-scheduling/common/policy.libsonnet

```diff
@@ -83,7 +83,7 @@ function(cfg, params={}, metadata={}) {
       + spec.v1.AdaptiveLoadScheduler.withDryRun(updatedConfig.policy.service_protection_core.dry_run)
       + spec.v1.AdaptiveLoadScheduler.withInPorts({
         overload_confirmation: (if isConfirmationCriteria then spec.v1.Port.withSignalName('OVERLOAD_CONFIRMATION') else spec.v1.Port.withConstantSignal(1)),
-        signal: spec.v1.Port.withSignalName('SIGNAL'),
```

Comment:
This is going to break the other variants since they may be based on an arbitrary signal from promql? Instead change the signature of `function` in the common code to take the names of signal and setpoint signals so that they may be customized according to the variant.

Reply:
you are right, thanks for noticing